boards/arm/rtl8762gn_evb/rtl8762gn_evb.dts

```diff
@@ -0,0 +1,55 @@
+/dts-v1/;
+#include <rtk/rtl8762gn_evb.dtsi>
+#include "rtl8762gn_evb-pinctrl.dtsi"
+
+/ {
+	model = "RealTek rtl8762gn evb ";
+	compatible = "rtk,rtl8762gn_evb";
```

Comment:
Please change the "rtk,rtl8762gn_evb" to"realtek,rtl8762gn_evb"

Reply:
Done, please help review